src/main/java/io/vertx/core/eventbus/impl/ReplyHandler.java

```diff
@@ -70,7 +70,7 @@ protected void doReceive(Message<T> reply) {
   }
 
   void register() {
-    register(repliedAddress, true, ar -> {});
+    register(repliedAddress, true, Promise.promise());
```

Comment:
we should pass `null` and allow the downstream `EventBusImpl#addRegistration` to be passed `null`, to avoid creating a `Promise` when it is not needed.

Reply:
done